From: "Barak A. Pearlmutter" <barak+git@cs.nuim.ie>
Date: Thu, 16 Oct 2014 08:36:28 +0100
Subject: disable ivgetjpg

The ivgetjpg script relies on the absent program tmpnam, and in any
case other ways of achieving its effect exist.  I.e., the program
would be fixable, but it's not worth it.

See https://bugs.debian.org/323480
---
 Makefile.am | 6 +-----
 1 file changed, 1 insertion(+), 5 deletions(-)

diff --git a/Makefile.am b/Makefile.am
index 92cc3b7..bfd9b9b 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -659,11 +659,7 @@ InterViews: src/IV/InterViews.defaults
 
 ### binaries
 
-bin_SCRIPTS = ivgetjpg ivtiftopnm pnmtopgm
-
-ivgetjpg: src/scripts/ivgetjpg.bash
-	cp $< $@
-	chmod a+x $@
+bin_SCRIPTS = ivtiftopnm pnmtopgm
 
 ivtiftopnm: src/scripts/ivtiftopnm.bash
 	cp $< $@
